assets/szn.js

```diff
@@ -0,0 +1,47 @@
+var chance = 5;
```

Comment:
would 1/2 chance be too mean? just want every visitor to have a chance to see this art 😈

Reply:
oof maybe I could definitely show it the first time and make it a chance after that?